### PR TITLE
add visibleItemCount property for datetime-picker

### PR DESCRIPTION
**Pull request by hstarorg**
datetime-picker can not set visible-item-count, but I need this feature.

vue-msgbox need "babel-preset-stage-2", so I add it in package.json

### CODE REVIEW

File: packages/datetime-picker/src/datetime-picker.vue [hunk 1]
```diff
@@ -1,51 +1,46 @@
...
-      :visible-item-count="7"
-      class="mint-datetime-picker"
-      v-ref:picker
-      show-toolbar>
-      <span class="mint-datetime-action mint-datetime-cancel" @click="visible = false">{{ cancelText }}</span>
-      <span class="mint-datetime-action mint-datetime-confirm" @click="confirm">{{ confirmText }}</span>
-    </mt-picker>
-  </mt-popup>
+	<mt-popup :visible.sync="visible" position="bottom" class="mint-datetime">
+		<mt-picker
```
Comment (QingWei-Li) [inline packages/datetime-picker/src/datetime-picker.vue]: two spaces

**Comment by hstarorg** [comment]
fixed

File: packages/datetime-picker/src/datetime-picker.vue [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-    <mt-picker
-      :slots="dateSlots"
-      @change="onChange"
-      :visible-item-count="7"
-      class="mint-datetime-picker"
-      v-ref:picker
-      show-toolbar>
+    <mt-picker 
+    :slots="dateSlots" 
+    @change="onChange"
```
Comment (QingWei-Li) [inline packages/datetime-picker/src/datetime-picker.vue]: Please use two spaces, and don't have whitespace at the end  😂

**Review by QingWei-Li — APPROVED** [review]

**Comment by souldjl** [comment]
大佬们，这个在苹果手机 ，只能显示一个 item 是为什么啊

**Pull request merged**